deploy/terraform/mod/core/main.tf

```diff
@@ -18,7 +18,7 @@ resource "kubernetes_stateful_set" "core_stateful_set" {
   }
   spec {
     service_name = "core-agents"
-    replicas = 1
+    replicas     = 1
```

Comment:
what tool do you use to link these files? I installed a plugin for clion, but I guess it must not do linting.

Reply:
terraform has a format command :) I have a nvim plugin that allows me to specify what formatted to use per file, and so for me it's a matter of `,fm` -- where `,` leader key